tests/e2e/main.go

```diff
@@ -224,7 +224,7 @@ var rootCmd = &cobra.Command{
 		apiextensionsscheme.AddToScheme(scheme.Scheme)
 		decode := scheme.Codecs.UniversalDeserializer().Decode
 		// create rbac
-		if rbacData, err := ioutil.ReadFile(filepath.Join(assetsPath, "manifests/rbac.yaml")); err != nil {
+		if rbacData, err := ioutil.ReadFile(filepath.Join(assetsPath, "manifests", "rbac.yaml")); err != nil {
```

Comment:
Create a utility function:

    func readManifestFile(filename string) ... {
        ... ioutil.ReadFile(filepath.Join(assetsPath, "manifests", filename)
        ...
    }

that way you don't have to worry about typos in "manifests" and easier to change in place should the location change over time.